### PR TITLE
Update Troubleshooting page with instructions to deploy Kubecost without persistent storage

### PR DESCRIPTION
**Pull request by linhlam-kc**
Update Troubleshooting page with step by step instructions to deploy Kubecost without persistent storage for experimental purpose. It could help customer to quickly troubleshoot or test Kubecost deployment on K8s cluster that has restricted persistent storage.

### CODE REVIEW

File: troubleshoot-install.md [hunk 1]
```diff
@@ -50,7 +50,21 @@ If you don’t see a name, you need to add a storage class. For help doing this,
...
+1. On your terminal, run this command to add the kubecost helm repository:
+```helm repo add kubecost https://kubecost.github.io/cost-analyzer/```
+
+1. Next, run this command to deploy Kubecost without persistent storage:
+<pre>
+helm upgrade -i --create-namespace kubecost kubecost/cost-analyzer \
+--namespace kubecost \
+--set persistentVolume.enabled="false" \
+--set prometheus.server.persistentVolume.enabled="false" \
+--set kubecostToken="aGVsbUBrdWJlY29zdC5jb20=xm343yadf98"
```
Comment (jessegoodier) [inline troubleshoot-install.md:66]: maybe just say <your key> here.

**Comment by jessegoodier** [comment]
NIce first PR on your 2nd day! I think that is a record, maybe by a large margin. 

one edit and I'd say you should merge>squash>delete branch

File: troubleshoot-install.md (hunk 1 above)
Comment (dwbrown2) [reply troubleshoot-install.md:66]: Yep, or drop token all together? I think it would reuse on `upgrade`

File: troubleshoot-install.md [hunk 2]
```diff
@@ -50,7 +50,21 @@ If you don’t see a name, you need to add a storage class. For help doing this,
 * AWS: [https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
 * Azure: [https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk](https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk)
 
-Alternatively, you can [deploy Kubecost without persistent storage](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml#L148).
+Alternatively, you can deploy Kubecost without persistent storage to store by following these steps:
+
+> Note: This setup is only for experimental purpose. The metric data is reset when kubecost's pod is rescheduled.
```
Comment (dwbrown2) [inline troubleshoot-install.md:55]: Potential Future TODO: we may want to reference our new backup & restore feature here... i.e. data is only reset when external storage isn't enabled.

File: troubleshoot-install.md (hunk 1 above)
Comment (dwbrown2) [reply troubleshoot-install.md:66]: Yeah, or can we just drop the `kubecostToken` line? I believe helm will reuse on `upgrade`

**Review by dwbrown2 — APPROVED** [review]
Killer stuff! Welcome to the team Lihn :)

File: troubleshoot-install.md (hunk 1 above)
Comment (linhlam-kc) [reply troubleshoot-install.md:66]: Thank you Jesse, Webb. I will drop it in final edit. I wasn't sure if it is required, so I add it to match with what we have in README of Helm chart repository.

File: troubleshoot-install.md (hunk 2 above)
Comment (linhlam-kc) [reply troubleshoot-install.md:55]: Noted. Thank you for your feedback Webb, I will try new feature and update the doc accordingly in new PR.

File: troubleshoot-install.md [hunk 3]
```diff
@@ -50,7 +50,20 @@ If you don’t see a name, you need to add a storage class. For help doing this,
...
+
+1. Next, run this command to deploy Kubecost without persistent storage:
+<pre>
+helm upgrade -i --create-namespace kubecost kubecost/cost-analyzer \
+--namespace kubecost \
+--set persistentVolume.enabled="false" \
+--set prometheus.server.persistentVolume.enabled="false"
+</pre>
 
 ## <a name="port-forward"></a>Issue: unable to establish a port-forward connection
```
Comment (jessegoodier) [inline troubleshoot-install.md:68]: do you mind removing all the anchors?
 example: "<a name="port-forward"></a>" 
from all sections?
we found out that it messes up the table of contents

I realize I was probably the one that added it.

**Pull request merged**